packages/speech/src/index.js

```diff
@@ -432,11 +435,19 @@ Speech.prototype.createRecognizeStream = function(config) {
     throw new Error('A recognize request requires a configuration object.');
   }
 
-  config = extend(true, {
-    config: {
-      languageCode: 'en-US'
-    }
-  }, config);
+  // As of Speech v1, a language code is required; throw an exception if we
+  // did not receive one.
+  //
+  // This is expected within a nested config, but in the interest of user
+  // sanity, accept it in the outer config object.
+  config.config = config.config || {};
+  if (config.languageCode) {
+    config.config.languageCode = config.languageCode;
+    delete config.languageCode;
+  }
+  if (is.undefined(config.config.languageCode)) {
+    throw new Error('A `languageCode` is required in the config object.');
```

Comment:
Hmm.

Not entirely true. The other client libraries have hard language code requirements, because we made `language_code` or `languageCode` a required argument to the methods. In other words, the Python or Ruby client will never _get to_ the server because you would get an error message saying, "`recognize` has one required argument, you sent zero" (paraphrase).

At minimum, the notion that you can relax this in the future and most clients will require no changes is untrue. It is technically feasible for _this_ one since all arguments in JavaScript are optional.